saleor/graphql/shipping/types.py

```diff
@@ -327,8 +335,28 @@ def resolve_id(root: ChannelContext, _info):
 
     @staticmethod
     def resolve_active(root: ChannelContext, _info):
+        # Currently selected shipping method is not validated
+        # with webhooks on every single API call
+        if not hasattr(root.node, "active"):
+            return True
         return root.node.active
 
     @staticmethod
     def resolve_message(root: ChannelContext, _info):
+        # Currently selected shipping method is not validated
+        # with webhooks on every single API call
+        if not hasattr(root.node, "message"):
+            return True
```

Comment:
It should return an empty string in this case I guess? Btw. I might have missed it earlier, but we have this `description` field which is used for providing descriptions to shipping methods. Couldn't we use it instead of `message`? Description is using Editor.js format but it's a pretty simple JSON - see `dummy_editorjs` fixture for example.